### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by javaguirre** [ddavison/rest-client#9]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/usr/share/atom/resources/app/src/atom.js:55:11)
Object.activate (/home/javaguirre/.atom/packages/rest-client/lib/rest-client.coffee:18:9)
```

**Comment by javaguirre** [comment]
I'll take a look at that today.

**Comment by ddavison** [comment]
fixed in 6b74d29ed79e858a5c4de35d4e8bd7f3854f5509
